packages/pds-ember/addon/components/pds/popup/index.hbs

```diff
@@ -0,0 +1,32 @@
+<div
+  class="pds-popup pds-popup--{{this.variant}}"
+  ...attributes
+  data-test-popup
+>
+  <div
+    class="pds-popup__icon"
+    data-test-popup-icon
+  >
+    <Pds::Icon @type={{this.iconType}} />
+  </div>
+
+  <div
+    class="pds-popup__close"
+    data-test-popup-close
+  >
+    <Pds::Button
+      @compact={{true}}
+      @iconStart="cancel-plain"
+      @variant="ghost"
+      {{on 'click' this._handleClose}}
+      aria-label="close popup"
```

Comment:
Should this be an arg or a getter with a default value so that it can be overridden with something that's i18n'd?

Reply:
According to `ember-intl` [documentation](https://ember-intl.github.io/ember-intl/docs/advanced/addon-support#overriding-translations), translations can be overridden by the host app, so there should be no need to define an argument.  However, i18n hasn't been fully implemented across components yet, so this component will be updated when we apply translations across the addon.

Reply:
FYI: PDS-252 has been created to capture this work.